_posts/2021-12-31-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,348 @@
+* [Awesome Java](https://github.com/akullpp/awesome-java/pull/917)
+* [Awesome For Beginners](https://github.com/MunGell/awesome-for-beginners/pull/933)
+* [Airflow](https://github.com/apache/airflow/pull/15187)
+* [trino-gateway](https://github.com/lyft/presto-gateway/issues/134)
+* [Metabase](https://github.com/metabase/metabase/issues/17532)
+* and so much more...
+
+Not only was the community excited and supportive about the brand change, but
+particularly they were all loving the new mascot. Our adorable bunny was soon 
+after [named Commander Bun Bun by the community](/episodes/10.html).
+
+<p align="center">
+ <a href="https://twitter.com/jtannady/status/1346888143459545092" target="_blank">
+   <img align="center" width="50%" src="/assets/blog/2021-review/cbb.png"/>
+ </a>
+</p>
+
+Immediately after the announcement, the [Trino GitHub repository](https://github.com/trinodb/trino)
+started trending in number of stargazers. Up until this point, the PrestoSQL
+GitHub repository had only amassed 1,600 stargazers in the two years since it 
+had split from the PrestoDB repository. However, within four months after the 
+renaming, the number of stargazers had doubled. GitHub stars, issues, pull 
+requests and commits started growing at a new trajectory.
+
+<p align="center">
+ <a href="https://twitter.com/bitsondatadev/status/1344028682126565381" target="_blank">
+   <img align="center" width="50%" src="/assets/blog/2021-review/trending.png"/>
+ </a>
+</p>
+
+## Community growth
+
+While we are on the subject, lets talk about the health of the Trino community.
+As mentioned the GitHub stargazers trajectory skyrocketed. At the time of 
+writing, we just hit 4,600 stargazers on GitHub. This means, we have added over
+3,000 stargazers in the last year, a 187% increase.
+
+Aside from GitHub, we have seen a 47% growth of [the Trino Twitter](https://twitter.com/trinodb) 
+followers this year. [The Trino Slack community](https://trino.io/slack.html),
+where a large amount of troubleshooting and development discussions occur, saw a
+75% growth nearing 6,000 members. Finally, [the Trino YouTube channel](https://www.youtube.com/c/TrinoDB)
+has seen an impressive 280% growth in subscribers as the [Trino Community Broadcast](/broadcast/)
+has grown in the number of helpful videos for community members.
+
+While stargazers, subscribers, and followers tell the story of the growing 
+awareness of the Trino project with the new name, what about the actual rate of
+development on the project?
+
+At the start of the year, there were 21,924 commits. This year, we pushed 3,679 
+commits to the repository,sitting at over 25,600 now. Looking at the graph, this
+keeps us pretty consistent with 2020s throughput.
+
+<p align="center">
+ <img align="center" width="75%" src="/assets/blog/2021-review/commits.png"/>
+</p>
+
+With the project's trajectory displayed in numbers, let's examine the top 
+features that landed in Trino this year.
+
+## Features
```

Comment:
HTTP Event listener: https://trino.io/docs/current/admin/event-listeners-http.html

Reply:
Custom connector metrics

Reply:
Transparent query retries

Reply:
Column and table lineage tracking in QueryCompletedEvent